packages/docusaurus-theme-classic/src/theme/BlogListPage/index.js

```diff
@@ -21,16 +21,14 @@ function BlogListPage(props) {
           <div className="col col--8 col--offset-2">
             {items.map(
               ({content: BlogPostContent, metadata: blogPostMetadata}) => (
-                <div
```

Comment:
What's wrong with a wrapper div?

Reply:
It is not needed, we can insert the CSS class directly into the `article` tag, why an extra tag (div)?

Reply:
I mean, the `<div>` is there so that we can add margin. If we don't need to add margin of course not having a div is better.